plugins/UFPWriter/UFPWriter.py

```diff
@@ -151,8 +157,26 @@ def _writeObjectList(archive):
 
         To retrieve, use: `archive.getMetadata(METADATA_OBJECTS_PATH)`
         """
+
         objects_model = CuraApplication.getInstance().getObjectsModel()
-        object_metas = [{"name": item["name"]} for item in objects_model.items]
+        object_metas = []
+
+        for item in objects_model.items:
+            object_metas.extend(UFPWriter._getObjectMetas(item["node"]))
 
         data = {METADATA_OBJECTS_PATH: object_metas}
         archive.setMetadata(data)
+
+    @staticmethod
+    def _getObjectMetas(node: SceneNode) -> List[Dict[str, str]]:
```

Comment:
Might I suggest `_getObjectMetadata` instead? It is codestyle not to abbreviate, and a "meta" isn't a thing.

Reply:
Fixed in https://github.com/Ultimaker/Cura/pull/8023/commits/72310919c3748eb91832b1e5494161123a430e2d.